### PR TITLE
fix: add tests for feed row - vue3

### PR DESCRIPTION
**Pull request by sanjeevdialpad**
# Feed Row Tests

## :hammer_and_wrench: Type Of Change

- [x] Fix
- [ ] Feature
- [ ] Refactoring
- [ ] Documentation

## :book: Description

Add tests for feed item row and remove $listeners since its deprecated in vue3

## :pencil: Checklist

- [ ] I have reviewed my changes
- [ ] I have added tests
- [ ] I have added all relevant documentation
- [ ] I have validated components with a screen reader
- [ ] I have validated components keyboard navigation
- [ ] I have considered the performance impact of my change
- [ ] I have checked that my change did not significantly increase bundle size
- [ ] I am exporting any new components or constants in the index.js in the component directory
- [ ] I am exporting any new components or constants in the index.js in the root

### CODE REVIEW

File: recipes/conversation_view/feed_item_row/feed_item_row.test.js [hunk 1]
```diff
@@ -0,0 +1,189 @@
+import { mount } from '@vue/test-utils';
+import DtFeedItemRow from './feed_item_row.vue';
+
+const basePropsData = {
```
Comment (braddialpad) [inline recipes/conversation_view/feed_item_row/feed_item_row.test.js]: Rename to baseProps just for consistiency.

**Comment by braddialpad** [comment]
Make sure to cherry-pick your changes from the other PR over to this one.

**Comment by sanjeevdialpad** [comment]
will fix the unit test. Confused how it failed in vue3

**Comment by sanjeevdialpad** [comment]
@braddialpad done.

**Review by braddialpad — APPROVED** [review]

**Pull request merged**